sriovnet.go

```diff
@@ -382,3 +383,21 @@ func FreeVfByNetdevName(handle *PfNetdevHandle, vfIndex int) error {
 func GetVfNetdevName(handle *PfNetdevHandle, vf *VfObj) string {
 	return vfNetdevNameFromParent(handle.PfNetdevName, vf.Index)
 }
+
```

Comment:
an small example is good to have here in the exported API, specially with the VF name such as 0000:06:00:01 or just 06:00:01 is good enough.

Reply:
Done